langchain/document_loaders/googledrive.py

```diff
@@ -170,15 +171,11 @@ def _load_document_from_id(self, id: str) -> Document:
         """Load a document from an ID."""
         from io import BytesIO
 
-        from googleapiclient.discovery import build
         from googleapiclient.errors import HttpError
         from googleapiclient.http import MediaIoBaseDownload
 
-        creds = self._load_credentials()
-        service = build("drive", "v3", credentials=creds)
-
-        file = service.files().get(fileId=id, supportsAllDrives=True).execute()
-        request = service.files().export_media(fileId=id, mimeType="text/plain")
+        file = self._service().files().get(fileId=id, supportsAllDrives=True).execute()
```

Comment:
can we do `service = self._service()` and then use in both these lines?

Reply:
see above